base/strings/io.jl

```diff
@@ -54,8 +56,8 @@ end
 """
     println([io::IO], xs...)
 
-Print (using [`print`](@ref)) `xs` followed by a newline.
-If `io` is not supplied, prints to [`stdout`](@ref).
+Print (using [`print`](@ref)) to `io` (or to the default output stream [`stdout`](@ref)
+if `io` is not given) `xs` followed by a newline.
```

Comment:
IMO, this change makes the sentence a lot harder to read. Without the parentheses it reads “Print to `io` `xs`”, I would prefer “Print `xs` to `io`”.

Suggestion:
```suggestion
Print (using [`print`](@ref)) `xs` to `io` followed by a newline.
If `io` is not supplied, prints to the default output stream [`stdout`](@ref).
```